Manito/Manito/Presentation/Scene/DetailScene/Detail/View/Cell/FriendCollectionViewCell.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  FriendCollectionViewCell.swift
+//  Manito
+//
+//  Created by 최성희 on 2022/06/13.
```

Comment:
p 560012456;

```suggestion
//  Created by 신윤아 on 2023/10/22.
```

...ㅋㅋㅋ  그냥 바꾸면 어떨까 해서 달았어요~